frontend/src/scene/payment/PaymentView.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import styled from 'react-emotion';
+import Modal, { Title, Content as ModalContent } from '../../components/modal';
+import Button from '../../components/button';
+import posed from 'react-pose';
+import { connect } from 'utils';
+import { Link } from 'react-router-dom';
+import PaymentState from './state';
+import qs from 'qs';
+
+const Content = styled(ModalContent)`
+    width: 100%;
+    height: 100%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+    margin-top: 10rem;
+
+    div {
+        display: inherit;
+        flex-direction: column;
+        text-align: center;
+        margin: 4rem;
+
+        * {
+            margin: 1rem;
+        }
+        span {
+            font-size: 8rem;
+            font-family: GT-Walsheim, sans-serif;
+            color: ${(props) => props.theme.main};
+        }
+    }
+`;
+
+const PaymentInfoArea = posed.div({});
+
+class PaymentView extends Component {
+    paymentState = new PaymentState(this.props.userStore);
+    render() {
+        const params = this.props.params;
+        let parsed = null;
+
+        if (params) {
+            parsed = qs.parse(params.slice(1));
+        }
+        if (parsed !== null) {
```

Comment:
No need to explicitly set the `null` in the first place and this check can be simplified to be just `if (parsed)`. Also it could include the inner `if` condition so `if (parsed && parsed.orderNumber`. Which should be `orderNumber` as far as I can see.

Reply:
corrected